charts/mgmt-agent/Chart.yaml

```diff
@@ -18,7 +18,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.0.2
+version: 3.0.3
```

Comment:
do we need to update the version in the top level Chart.yaml?

Reply:
yes. its there in pending task. once I have the 1.4.0, I will bump the oci-onm version as well

Reply:
done